src/spdx_tools/spdx/parser/tagvalue/parser.py

```diff
@@ -180,7 +180,7 @@ def p_current_element_error(self, p):
         "file_comment : FILE_COMMENT text_or_line\n "
         "file_license_concluded : FILE_LICENSE_CONCLUDED license_or_no_assertion_or_none\n "
         "package_name : PKG_NAME LINE\n description : PKG_DESCRIPTION text_or_line\n "
-        "summary : PKG_SUMMARY text_or_line\n source_info : PKG_SOURCE_INFO text_or_line\n "
+        "summary : PKG_SUMMARY text_or_line\n source_info : PKG_SOURCE_INFO text_or_line_including_no_assertion\n "
```

Comment:
The problem at hand can also occur for other `text_or_line` fields apart from `source_info`. Basically all properties that use `text_or_line`. So I would believe that the whole `text_or_line` property has to be adapted, not only for the source information case.

Reply:
Also, the same problem will arise when somebody uses `NONE`. So best solve this one here, too.